go-controller/pkg/ovn/egressip_test.go

```diff
@@ -622,7 +623,7 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 				key2 := node2.Name
 				gomega.Eventually(func() *retryObjEntry {
 					return fakeOvn.controller.retryEgressNodes.getObjRetryEntry(key2)
-				}).ShouldNot(gomega.BeNil())
```

Comment:
hm.... I hit this when running the PR on GH (below).
Maybe we should use 4 instead of 2 as the value for `inspectTimeput`?
```
 [Fail] OVN master EgressIP Operations On node UPDATE [It] using EgressNode retry should re-assign EgressIPs and perform proper OVN transactions when pod is created after node egress label switch 
/home/runner/work/ovn-kubernetes/ovn-kubernetes/go-controller/pkg/ovn/egressip_test.go:626
[Fail] OVN master EgressIP Operations On node UPDATE [It] should re-assign EgressIPs and perform proper OVN transactions when namespace and pod is created after node egress label switch 
/home/runner/work/ovn-kubernetes/ovn-kubernetes/go-controller/pkg/ovn/egressip_test.go:984
```

Reply:
Yeah, let's try with 4. I've seen it fail with 2 already... :-/